.github/CONTRIBUTING.rst

```diff
@@ -0,0 +1,50 @@
+Contributing to Devguide
+========================
```

Comment:
How about converting this file to Markdown as well? For consistency with the other GitHub community files.

Reply:
Because the rest of the entire repo is reST and adding a single Markdown adds an extra requirement to contributing to the repo (and trust me, you can't assume people know Markdown even if they happen to know reST).